plugins/source/github/client/services.go

```diff
@@ -15,6 +15,7 @@ type GithubServices struct {
 	Repositories    RepositoriesService
 	Teams           TeamsService
 	DependencyGraph DependencyGraphService
+	Users           UsersService
```

Comment:
You need to initialize the `Users` service via https://github.com/cloudquery/cloudquery/blob/e4d94d08c9ebebf08ded2a7b1b33fe5bcd0976c4/plugins/source/github/client/client.go#L164